### DISCUSSION TITLE
How can I make nice looking tables for pdf output?

### DISCUSSION THREAD

**Discussion by e05bf027** [quarto-dev/quarto-cli#6314] [Q&A] [answered]
### I want to create prettier tables manually

I find Quarto to be generally excellent, but I am really struggling to produce pdfs with tables in them that look good. I have tried using the `booktabs` package and I am pretty certain I have given correct instructions for it to be loaded it correctly. I keep getting error messages. I don't love the output from the tables created in the Visual Editor, and want more control over formatting.

So I suppose my question is: does anyone know how to make tables manually that look decent in pdf output?

**I am using Quarto, with R, in an M1 Macbook Air**

**Comment by mcanouil** [comment, marked as answer]
Either use libraries/packages outputting LaTeX code for the tables.
Use raw LaTeX `` ```{=latex} `` and write your own LaTeX code.
And if you struggle with your LaTeX distribution, you should really consider using TinyTex which can be installed by Quarto (see the documentation).

**Reply by e05bf027** [reply]
It is all working now as I has hoped it would! I installed TinyTex, and wrote raw Latex (which I thought I was doing, but by your post could see that I was not doing it correctly).

Thank you.
